packages/flutter_tools/test/commands.shard/hermetic/analyze_test.dart

```diff
@@ -52,3 +69,47 @@ bool inRepo(List<String> fileList, FileSystem fileSystem) {
   }
   return false;
 }
+
+class MockAnalyze extends AnalyzeBase {
```

Comment:
See comment about making `generateDartDocMessage` static/top level - then this class isn't needed